### ISSUE TITLE
Accessibility for selecting the search Bar

### ISSUE THREAD

**Issue by amyria3** [amyria3/political#16]
- [ ]  When selecting Search with TAB, searchBar should have the same optics like on Hover

- [ ] Add the label oder aria-label oder aria-labelledby: Wenn du das label-Element nicht verwenden kannst, kannst du alternativ das aria-label-Attribut verwenden, um eine beschreibende Beschriftung bereitzustellen
- [ ] Role="search": Verwende das "search"-Rollenattribut, um anzuzeigen, dass es sich um ein Suchfeld handelt

**Comment by amyria3** [comment]
Feel free to comment
